cmd/podman/images/scp.go

```diff
@@ -374,8 +347,124 @@ func parseArgs(args []string, cfg *config.Config) (map[string]config.Destination
 				return nil, err
 			}
 		}
-		scpOpts.URI = append(scpOpts.URI, urlT)
-		scpOpts.Iden = append(scpOpts.Iden, iden)
+		sshInfo.URI = append(sshInfo.URI, urlT)
+		sshInfo.Identities = append(sshInfo.Identities, iden)
 	}
 	return serv, nil
 }
+
+// execLoad executes the podman load command given the podman binary
+func execLoad(podman string) error {
+	var loadCommand []string
+	if source.Quiet {
```

Comment:
Simplify this into one one line.
quiet:=""
if source.Quiet {
       quiet="-q "
}
loadCommand = strings.Split(fmt.Sprintf(load %s--input %s, quiet, dest.File), " ")